content/reference-capabilities/capability-matrix.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Reference Capability Matrix"
+section: "Reference Capabilities"
+menu:
+  toc:
+    parent: "reference-capabilities"
+    weight: 110
+toc: true
+---
+
+At this point, it's quite possible that you read the previous sections in this chapter and are still pretty confused about the relation between reference capabilities. It's okay! We have all struggled when learning this part of Pony, too. Once you start working on Pony code, you'll get a better intuition with them.
+
+In the meantime, if you still feel like all these tidbits in the chapter are still scrambled in your head, there is one resource often presented with Pony that can give you a more visual representation: the __reference capability matrix__.
+
+It is also the origin of the concept behind each capability in Pony, in the sense of how each capability denies certain properties to its reference -- in other words, which guarantees a capability makes. We will explain what that actually means before presenting the matrix.
+
+## Local and global aliases
+
+Before anything else, we want to clarify what we mean by "local" and "global" aliases.
+
+A local alias is a reference to the same variable that exists in the same actor. Whenever you pass a value around, and it's not the argument of an actor's behavior, this is the kind of alias we are working with.
+
+On the other hand, a global alias is a reference to the same variable that can exist in a _different_ actor. That is, it describes the properties of how two or more actors could interact with the same reference.
+
+Each reference capability in Pony is actually a pair of local guarantees and global guarantees. For instance, `ref` doesn't deny any read/write capabilities inside the actor, but denies other actors from reading or writing to that reference.
+
+You may recall from the [Reference Capability Guarantees](guarantees.md) section that mutable references cannot be safely shared between actors, while immutable references can be read by multiple actors. In other words, this implies that global properties are as restrictive or more restrictive than the local properties to that reference; it's not possible to write to an immutable reference in a local alias, for example.
```

Comment:
This paragraph still seems a bit scattered to me. I've tried to suggest some changes that make it a bit more digestible, in my opinion:

```suggestion
You may recall from the [Reference Capability Guarantees](guarantees.md) section that mutable references cannot be safely shared between actors, while immutable references can be read by multiple actors. In general, global properties are always as restrictive or more restrictive than the local properties to that reference - what is denied globally must also be denied locally. For example, it's not possible to write to an immutable reference in either a global or local alias.
```

Reply:
I have added your change, but I'll review this paragraph myself later.